EKF/common.h

```diff
@@ -223,7 +223,7 @@ struct parameters {
 
 	// process noise
 	float gyro_bias_p_noise{1.0e-3f};	///< process noise for IMU rate gyro bias prediction (rad/sec**2)
-	float accel_bias_p_noise{6.0e-3f};	///< process noise for IMU accelerometer bias prediction (m/sec**3)
+	float accel_bias_p_noise{1.0e-2f};	///< process noise for IMU accelerometer bias prediction (m/sec**3)
```

Comment:
The default parameter in PX4 for the accel bias process noise is 3e-3 ([EKF2_ACC_B_NOISE](https://dev.px4.io/master/en/advanced/parameter_reference.html#EKF2_ACC_B_NOISE)). Here, you then multiply the process noise by a factor of 3 to fix the initial stability issue of the bias. 
Isn't the initial divergence a sign of something going wrong? I agree that increasing the process noise accelerate convergence before takeoff, but the bias shouldn't go initially in the wrong direction.

Reply:
Agree, but we are doing those covariance calculations in single precision and stability issues when the variances for those states become very small has been something we have encountered before.

I have some ideas about how we can further investigate this which we can discuss at the dev call. These include instrumenting all the fusion methods active during replay of this log and identifying all those that drive the bias state positive. Also we need to fix the bug that stops replay logging before 26 seconds.